### ISSUE TITLE
Legend UI bug for ORFrame Connector

### ISSUE THREAD

**Issue by sa-mm** [emeeks/semiotic#52]
I think this is actually a UI bug, as opposed to something I'm misunderstanding.

When I use a legend of type 'line' with ORFrame of type 'point', the transform applied to the `legend-item` group is `translate(0,45)`. The legend divider touches the first diagonal legend line in the legend.

You can see this in the first svg linked below. If I change the transform to `translate(0,55)`, then you get the second svg linked below, which looks OK to me.

https://repl.it/KGGu/6

Let me know if you want me to put together a minimal working example.

**Comment by emeeks** [comment]
Yes please do, I wouldn't be surprised if there were some formatting issues at play with Legend as we've only used it in particular cases.

**Comment by sa-mm** [comment]
This should do it:

```
import React from "react";
import { render } from "react-dom";
import { Legend } from "semiotic";

const lineLegendGroups = [
  {
    type: "line",
    styleFn: d => ({ stroke: d.color }),
    items: [
      { label: "Line 1", color: "red" },
      { label: "Line 2", color: "blue" }
    ]
  }
];

const App = () => (
  <svg style={{ height: 200, width: 200 }}>
    <Legend title={"Test Line Legend"} legendGroups={lineLegendGroups} />
  </svg>
);

render(<App />, document.getElementById("root"));
```

**Comment by emeeks** [comment]
This will be fixed in 1.1.3
